content/00.build/90.contributing-to-documentation/20.contribution-guidelines.md

```diff
@@ -7,32 +7,44 @@ description: Learn how to contribute to zkSync Docs
 
+[github-draft-status]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft
+
 ## Commit conventions
 
 This project uses [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standards.
-For changes that are **code related**, use the `fix:`, `feat:`, or `chore:` tags in your commits.
-For typo or document related changes, please use the `docs:` tag.
+For changes that are **code related**, use the `chore:`, `fix:`, `feat:`, or `docs:` tags in your commits.
+
+- `chore` tagged commits will not create a release. Use for changes that do not directly affect the end user.
+- `fix` commits will create a patch release.
+- `feat` commits will create a minor release.
+- `docs` commits will create a minor release. Use for typo fixes or document related changes.
 
 ```sh
 git commit -m "docs: fix typo in guide"
 ```
 
 ### Signed git commits
 
-Your git commits need to be [signed with a verified signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
+Your git commits need to be [signed with a verified signature][github-verified-signature-commits].
+
+1. Follow the instructions to [generate a signing key][github-generate-signing-key].
+1. [Add the key to your GitHub account][github-add-key].
+1. [Tell git about your signing key][github-signing-key].
 
-1. Follow the instructions to
-  [generate a signing key](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key).
-1. [Add the key to your GitHub account](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account).
-1. [Tell git about your signing key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key).
+[github-verified-signature-commits]: https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification
+[github-generate-signing-key]: https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key
+[github-add-key]: https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account
+[github-signing-key]: https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key
 
 ## Contribution workflow
 
-To set up and run the project locally, consult the root [`README.md`](%%zk_git_repo_zksync-docs%%?tab=readme-ov-file#-zksync-developer-documentation) file.
+To set up and run the project locally, consult the root [`README.md`][project-readme] file.
+
+[project-readme]: %%zk_git_repo_zksync-docs%%?tab=readme-ov-file#-zksync-developer-documentation
```

Comment:
this link doesn't work for me

Reply:
Check it in the preview URL to see if it works. The reason it doesn't work directly in the markdown file is that it is using the zk_parser I made that you can find in `docs-nuxt-template`: https://github.com/matter-labs/docs-nuxt-template/blob/main/assets/_zksync.json

On build, the process will search for these tags and replace them with the proper string defined in the `_zksync.json` file.

Reply:
got it thank you for explanation, works in preview!